Updating DNS config call out (#5926)
cloudflare/cloudflare-docs: content/cloudflare-one/applications/configure-apps/_index.md

* [**SaaS applications**](/cloudflare-one/applications/configure-apps/saas-apps/) consist of applications your team relies on that are not hosted by your organization. Examples include Salesforce and Workday. To secure SaaS applications, you must integrate Cloudflare Access with the SaaS application's SSO configuration.

> * [**Self-hosted applications**](/cloudflare-one/applications/configure-apps/self-hosted-apps/) consist of internal applications that you host in your own environment. These can be the data center versions of tools like the Atlassian suite or applications created by your own team. To secure self-hosted applications, you must use Cloudflare's authoritative DNS (partial [CNAME configurations](/dns/zone-setups/partial-setup/)  may also be used) and [connect the application](/cloudflare-one/connections/connect-apps/) to Cloudflare.

Reviewer: ```suggestion
* [**Self-hosted applications**](/cloudflare-one/applications/configure-apps/self-hosted-apps/) consist of internal applications that you host in your own environment. These can be the data center versions of tools like the Atlassian suite or applications created by your own team. To secure self-hosted applications, you must use Cloudflare's DNS ([full setup](/dns/zone-setups/full-setup/) or [partial CNAME setup](/dns/zone-setups/partial-setup/)) and [connect the application](/cloudflare-one/connections/connect-apps/) to Cloudflare.
```

> * [**Self-hosted applications**](/cloudflare-one/applications/configure-apps/self-hosted-apps/) consist of internal applications that you host in your own environment. These can be the data center versions of tools like the Atlassian suite or applications created by your own team. To secure self-hosted applications, you must use Cloudflare's authoritative DNS (partial [CNAME configurations](/dns/zone-setups/partial-setup/)  may also be used) and [connect the application](/cloudflare-one/connections/connect-apps/) to Cloudflare.

Reviewer 2: ```suggestion
* [**Self-hosted applications**](/cloudflare-one/applications/configure-apps/self-hosted-apps/) consist of internal applications that you host in your own environment. These can be the data center versions of tools like the Atlassian suite or applications created by your own team. To secure self-hosted applications, you must use [Cloudflare as a DNS provider](/dns/zone-setups/) and [connect the application](/cloudflare-one/connections/connect-apps/) to Cloudflare's edge.
```
